samples/isilon_v260_ops_410.yaml

```diff
@@ -5,15 +5,15 @@ metadata:
   namespace: test-isilon
 spec:
   driver:
-    # Config version for CSI PowerScale v2.3.0 driver
```

Comment:
Hi @rajkumar-palani if 4.12 support is going to be added, we need not have 4.10 as we support only n-1 openshift versions

Reply:
Hi @rajendraindukuri. OpenShift 4.12 has been removed from CSM 1.6 Plan.